tests/run_compile.sh

```diff
@@ -75,7 +75,10 @@ if [[ $ROCOTO = 'false' ]]; then
   submit_and_wait job_card
 else
   chmod u+x job_card
-  ./job_card
+  ( ./job_card 2>&1 1>&3 3>&- | tee err ) 3>&1 1>&2 | tee out
```

Comment:
Is this and the similar change in `run_test.sh` necessary? I ran RT with and without these changes and did not notice any difference in any of the log files.

Reply:
Unless I'm mistaken, you need to redirect via tee to see logs in the job's own log file. Otherwise it only goes to the out and err files. There is similar code (but less reliable) in the non-Rocoto portions of the script.

Reply:
At some point we should remove all rocoto code, we do not use it and do not test it. For parallel execution we use ecflow.

Reply:
I'll retest this. I know I added this change in two locations. Let me check if I really need them in both locations. This may take a whole day due to queue wait times.

Reply:
@DusanJovic-NOAA  - It is premature to remove Rocoto support since most people outside EMC and NCO have no experience with Rocoto. I can keep it working on Jet and Hera until that interferes with my official duties.

Reply:
@MinsukJi-NOAA I retested. Yes, the changes are necessary.  This is half of the bug fix for the critical bug in the workflow: the job deleted the out and err files Rocoto created in its `#SBATCH` lines, causing the rt_test.sh to conclude the test failed (because its grep failed). Now, Rocoto and the job_script each have their own log files, and this line ensures the logging goes to both.